### PR TITLE
[WIP] Redo Qiskit.org in nuxt.js

### PR DESCRIPTION
**Pull request by delapuente**
Closes qiskit/qiskit.org#258

### CODE REVIEW

**Comment by marinaaisa** [comment]
## Just a friendly reminder, I found these 2 things left to do that you may not noticed:

1)  All subpages for Qiskit's elements have black background instead of white background as original page has. 

### Your branch
Captura de pantalla 2019-10-24 a las 12 37 13

### Original page
Captura de pantalla 2019-10-24 a las 12 39 02

2) Links are not using `nuxt-link` yet.

**Comment by delapuente** [comment]
@marinaaisa this is ready for another look.

**Comment by delapuente** [comment]
You're right in the sense that the color scheme should be reviewed (for achieving minimum accessibility) and there would be redundant content in the landing page. However, this is part of the reunification plan of qiskit.org and community.qiskit.org, and also part of a langing page complete review so it is expected the redundancy will go.

**Comment by delapuente** [comment]
I moved it out of draft but please, do not merge yet.

File: static/images/qiskit-aqua-logo.svg [hunk 1]
```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (marinaaisa) [inline static/images/qiskit-aqua-logo.svg]: All these SVGs can be cleaned and minified, I'm very good at it. I have created a new issue to solve it in other moment: https://github.com/Qiskit/qiskit.org/issues/315

**Comment by delapuente** [comment]
This is merged now.

**Pull request closed without merging**